### PR TITLE
Modify scripts for updating kythe

### PR DESCRIPTION
**Pull request by ciera**
Modifies the scripts to allow us to push out our own versions of kythe.

Unfortunately, this does not seem to fix the problem where ErrorProne is not producing results. It does fix the problem with the kythe image no longer existing. I tested the new image locally (ErrorProne fails, but kythe made compilation units) and am currently pushing the image.

### CODE REVIEW

File: third_party/kythe/README.google [hunk 1]
```diff
@@ -1,5 +1,5 @@
-URL: https://github.com/google/kythe/tree/250bb910b0edb5ccf99971692f8068a66ba87eac
-Version: 250bb910b0edb5ccf99971692f8068a66ba87eac
+URL: https://github.com/google/kythe/tree/c9f42bfb7e5b8d33834232e181fcf6a4e8105629
```
Comment (collinw) [inline third_party/kythe/README.google]: Is this change intended?

Comment (ciera) [inline third_party/kythe/README.google]: Yes, I went ahead and reran upgrade_kythe as part of building the new image. This will update hte commit number that we are using. Looks like there were no changes to this part of the tree otherwise, but at leas it's the same version as the image.
